rllib/algorithms/algorithm_config.py

```diff
@@ -781,6 +776,53 @@ def update_from_dict(
 
         return self
 
+    def get_state(self) -> Dict[str, Any]:
```

Comment:
Add `@override(Checkpointable)` and subclass AlgorithmConfig from `Checkpointable`.

Reply:
Yeah. This looks sound now. I am so glad when the old stack is gone. Everything will become so clean then.